localflavor/nl/validators.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+import re
+
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.utils.encoding import smart_text
+from django.utils.translation import ugettext_lazy as _
+
+
+class NLZipCodeFieldValidator(RegexValidator):
+    """
+    Validation for Dutch zip codes.
+
+    .. versionadded:: TODO
+    """
+    error_message = _('Enter a valid zip code.')
+
+    def __init__(self):
+        super(NLZipCodeFieldValidator, self).__init__(regex='^\d{4} ?[A-Z]{2}$',
+                                                      message=self.error_message)
+
+    def __call__(self, value):
+        super(NLZipCodeFieldValidator, self).__call__(value)
+
+        if int(value[:4]) < 1000:
+            raise ValidationError(self.error_message)
+
+
+class NLSoFiNumberFieldValidator(RegexValidator):
+    """
+    Validation for Dutch Sofinummers.
+
+    TODO: consider renaming to BSN
```

Comment:
Yeah, I agree that it should be changed to BSN but let's do this as separate issue.